packages/rum/run-script.js

```diff
@@ -113,26 +113,26 @@ function serveE2e(servingPath, port) {
   return [staticServer, backendAgentServer]
 }
 
-function runJasmine(cb) {
-  const specFiles = ['test/node/*.node-spec.js']
-  const jrunner = new JasmineRunner({
-    projectBaseDir: PROJECT_DIR,
-    specDir: ''
+function runJasmine(specDir, transform, cb) {
+  const jrunner = new JasmineRunner()
+  /**
+   * Files are relative to spec directory
+   */
+  jrunner.loadConfig({
+    spec_dir: specDir,
+    spec_files: ['*.spec.js'],
+    helpers: transform ? ['../babel-helper.js'] : []
   })
   jrunner.onComplete(passed => {
     if (!passed) {
-      const err = new Error('Jasmine node tests failed.')
+      const err = new Error('Jasmine tests failed')
       // The stack is not useful in this context.
       err.stack = ''
       cb(err)
     } else {
       cb()
     }
   })
-
-  jrunner.showColors(true)
-  jrunner.addReporter(JasmineRunner.ConsoleReporter(jrunner))
```

Comment:
Just to make sure this is intentional, we don't need to add the console report manually?